cmake/ElektraCompiling.cmake

```diff
@@ -35,6 +35,16 @@ endif()
 
 
 #
+# check if -Wl,--version-script linker option is supported
+# TODO: darwin ld only supports -Wl,-exported_symbols_list
+#       + the file syntax is different
+#
+set(__symbols_file "${CMAKE_CURRENT_BINARY_DIR}/symbols.map")
+file(WRITE ${__symbols_file} "{ local: *; };\n")
+check_cxx_compiler_flag("" LD_ACCEPTS_VERSION_SCRIPT)
```

Comment:
check_cxx_compiler_flag is correct. But I've forgotten to copy the CMAKE_REQUIRED_FLAGS line